### PR TITLE
Fixes #36725 - Improve bulk-upload command's output

### PR DESCRIPTION
**Pull request by ofedoren**

### CODE REVIEW

File: lib/hammer_cli_foreman_openscap/scap_content.rb [hunk 1]
```diff
@@ -28,9 +28,20 @@ class BulkUpload < HammerCLIForemanOpenscap::CreateCommand
...
 
+      def execute
+        d = send_request
+        d.delete('errors') if d['errors'].empty?
+        if d['results'].any? || d['errors'].any?
+          print_data(d)
+        else
+          print_success_message(d)
+        end
+        HammerCLI::EX_OK
```
Comment (adamruzicka) [inline lib/hammer_cli_foreman_openscap/scap_content.rb:39]: Is it still ok if `d['errors'].any?`?

Comment (ofedoren) [reply lib/hammer_cli_foreman_openscap/scap_content.rb:39]: Well, sometimes they can be something that can be ignored? https://github.com/theforeman/foreman_openscap/blob/master/lib/foreman_openscap/bulk_upload.rb#L40

I mean, it seems that we could return both OK or ERR. If we return ERR, but some of the content was actually uploaded, it's kinda misleading if you don't read the output. But if you expect everything to be uploaded and expect OK, but you get OK even though not everything was uploaded, it's... still misleading, so in this case IDK :D

Comment (adamruzicka) [reply lib/hammer_cli_foreman_openscap/scap_content.rb:39]: I'd like us to be a bit more strict and go with ERR. If we're asked to upload something and we, for any reason, don't do that, I'd expect that to be not ok.

Comment (ofedoren) [reply lib/hammer_cli_foreman_openscap/scap_content.rb:41]: Thanks, updated.

**Review by pondrejk — APPROVED** [review]
Checked on patched stream-28, it is either error or regular output now:
```
#  hammer scap-content bulk-upload --type directory --directory /usr/share/xml/scap/ssg/content
Uploaded Scap Contents: 
 1) Id:                5
    Title:             firefox content
    Original Filename: ssg-firefox-ds.xml
 2) Id:                6
    Title:             rhel6 content
    Original Filename: ssg-rhel6-ds.xml
 3) Id:                7
    Title:             rhel7 content
    Original Filename: ssg-rhel7-ds.xml
 4) Id:                8
    Title:             rhel8 content
    Original Filename: ssg-rhel8-ds.xml

Scap Contents uploaded.

]#  hammer scap-content bulk-upload --type directory --directory /gg
Errors: 
  No such directory: /gg. Please check the path you have provided.
```

**Review by adamruzicka — APPROVED** [review]

**Comment by adamruzicka** [comment]
Thank you @ofedoren & @pondrejk !

**Pull request merged**